Samples/ProvisionDemoLab/azuredeploy.parameters.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newLabName": {
+      "value": "FabrikamDemoLab"
+    },
+    
+    // parameters to set lab policies

```

Comment:
No, comments are not supported in json.

From: Mithun Shanbhag [mailto:notifications@github.com]
Sent: Tuesday, March 22, 2016 8:12 AM
To: Azure/azure-devtestlab <azure-devtestlab@noreply.github.com>
Subject: Re: [azure-devtestlab] Change ARM templates to create lab to include VNET creation (#66)


In Samples/ProvisionDemoLab/azuredeploy.parameters.json<https://github.com/Azure/azure-devtestlab/pull/66#discussion_r57003731>:

> @@ -0,0 +1,90 @@

> +{

> +  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",

> +  "contentVersion": "1.0.0.0",

> +  "parameters": {

> +    "newLabName": {

> +      "value": "FabrikamDemoLab"

> +    },

> +

> +    // parameters to set lab policies

Do comments work in a json file?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-devtestlab/pull/66/files/f5ff17abc5d8b9baab006c4ef44f19c55617e637#r57003731>